person.rb

```diff
@@ -0,0 +1,25 @@
+# Person class
+class Person
+  def initialize(id, age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] Kindly add the getters and setters to the person class for prmission

### Requirement:
![image](https://github.com/ngalamac/oop-school-library/assets/50990925/539926e1-6e12-476d-8d16-3517980ab6c3)

Reply:
Hi @Shedrack-Sunday 
Thank you for your kind review, I really appreciate it.
- Please I will like to clarify that in the project requirements, the getters and setter methods are not for all the parameters, only for the mentioned ones. That is why I did not create parent_permission getters and setters
- Also, I do not understand when you said "It will be nice to have the exact link to the project not just the issue link : https://github.com/users/alexmedinasf/projects/5"

Thank you

Reply:
Hello @ngalamac ,

-  Firstly, the link optional change was an error, apologies for that.
- Secondly, `name` and `age` instance variables are the only ones that should be able to be modified that's why they are defined as `attr_accessors` (here both Getters and Setters are defined). `id` should only be able to be retrieved so only the Getter is defined
  
 ### Take a look at the requirement:
![image](https://github.com/ngalamac/oop-school-library/assets/50990925/9db29469-192a-481d-9783-de30cc058757)